src/testers.jl

```diff
@@ -126,9 +126,9 @@ function test_scalar(f, z; rtol=1e-9, atol=1e-9, fdm=_fdm, fkwargs=NamedTuple(),
         frule_test(f, (z, Δx); rtol=rtol, atol=atol, fdm=fdm, fkwargs=fkwargs, kwargs...)
         if z isa Complex
             # check that same tangent is produced for tangent 1.0 and 1.0 + 0.0im
-            @test isapprox(
-                frule((Zero(), real(Δx)), f, z; fkwargs...)[2],
-                frule((Zero(), Δx), f, z; fkwargs...)[2],
+            check_equal(
+                frule((Zero(), real(Δx)), f, z; fkwargs...)[2]::Number,
+                frule((Zero(), Δx), f, z; fkwargs...)[2]::Number,
```

Comment:
scalar rules shout return `Numbers`.
but maybe this should be moved out of this part of the test and be somewhere else
tested explictly.

Reply:
I think we can leave this for now, I have removed this from the test